cmake/SSGCommon.cmake

```diff
@@ -1261,9 +1261,14 @@ macro(ssg_define_guide_and_table_tests)
         endforeach()
         add_test(
             NAME "unique-cces"
-	    COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/tests/assert_cces_unique.sh"
+	    COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/tests/assert_reference_unique.sh" "cce"
```

Comment:
Please don't mix tabs and spaces.

Reply:
I'm not mixing them. It was like this before. Anyway, I'll fix both indentation issues